WordPress/src/main/java/org/wordpress/android/ui/sitecreation/domains/SiteCreationDomainsViewModel.kt

```diff
@@ -140,7 +142,16 @@ class SiteCreationDomainsViewModel @Inject constructor(
             updateUiStateToContent(query, Loading(Ready(emptyList()), false))
             fetchDomainsJob = launch {
                 delay(THROTTLE_DELAY)
-                val onSuggestedDomains = fetchDomainsUseCase.fetchDomains(query.value, segmentId)
+                val onSuggestedDomains: OnSuggestedDomains
+                if (homePagePickerFeatureConfig.isEnabled()) {
```

Comment:
I added this check-in place just to support the segments if we decide to A/B test it still